src/effects/VST3/VST3Utils.cpp

```diff
@@ -128,13 +128,13 @@ bool VST3Wrapper::StoreSettings(const VST3EffectSettings& vst3settings) const
    if (!vst3settings.mProcessorStateStr)
       return false;
 
-   auto processorState = PresetsBufferStream::fromString(vst3settings.mProcessorStateStr.value());
+   auto processorState = PresetsBufferStream::fromString(*vst3settings.mProcessorStateStr);
    if (mEffectComponent->setState(processorState) != kResultOk)
       return false;
 
    if (vst3settings.mControllerStateStr)
    {
-      auto controllerState = PresetsBufferStream::fromString(vst3settings.mControllerStateStr.value());
+      auto controllerState = PresetsBufferStream::fromString(*vst3settings.mControllerStateStr);
```

Comment:
You anticipated me.  I see this now, taking the commits in sequence.  Good:  but of course rebase interactively and squash the fixes onto earlier commits before this is merged.